ui/appui-react/src/appui-react/hooks/useFrameworkVersion.tsx

```diff
@@ -28,14 +37,15 @@ export interface FrameworkVersionProps {
  * component uses the property frameworkState.configurableUiState.frameworkVersion from the redux store
  * to determine the ui version. This version will default to "2" and should only be set to "1" for older
  * iModelApp applications.
+ * @deprecated Used to toggle between UI1.0 and UI2.0.
  * @public
  */
-export function FrameworkVersion(props: FrameworkVersionProps) { // eslint-disable-line @typescript-eslint/no-redeclare
+export function FrameworkVersion(props: FrameworkVersionProps) { // eslint-disable-line @typescript-eslint/no-redeclare, deprecation/deprecation
```

Comment:
nit: redeclare should not be needed here.

Reply:
Good catch, removed.